docs/UPGRADING.md

```diff
@@ -2,6 +2,15 @@
 
 Here you can find a list of migration guides to handle breaking changes between releases of the CLI.
 
+## Unreleased
+
+### `board attach` CLI command and `BoardAttach` gRPC interface command removal
+
+Both the `board attach` CLI command and the `cc.arduino.cli.commands.v1.BoardAttach` gRPC command have been removed.
+
+They are not replaced by any new command, the only way to get the same behaviour is to edit the `sketch.json` file either manually or automatically with other tools.
```

Comment:
Is it true? is `sketch.json` supported in any way after removing `board attach`?